server/db/db.js

```diff
@@ -84,10 +57,45 @@ async function execute(path, params = {}) {
   sql = sql.replace(/\$\{[^{}]+\}/g, queryParam);
   const values = queryVariables ? queryVariables.map(p => params[p]) : [];
   return query(sql, values);
-}
+};
+
+const migrate = async () => {
```

Comment:
similar to the above, it'll help to add some jsdocs here describing what this function does